Tests/Core/Formatter/MultilineFormatterTests.swift

```diff
@@ -66,6 +66,35 @@ class MultilineFormatterTests: XCTestCase {
                          ]
                          """)
     }
+    
+    func testTupleString() {
+        let tupleElements: [(String?, String)] = [
```

Comment:
What time first argument of tuple is `nil`?
(Please write to test that passed to `nil` if possible)

Reply:
I can not find a situation in which the label is `nil`.

# Example

## the tuple is `()`

a formatter receives `[]`.

## no label, one element

the tuple is transformed to the inner element.

( `(#""one""#)` becomes `"one"` )

## no label element and labeled element

`(#""one""#, second: 2)` becomes `[(".0", "\"\"one\"\""), ("second", "2")]`.